### PR TITLE
Changes to remove style attributes with same properties from Element Web (Desktop)

### PR DESCRIPTION
**Pull request by JeanPaulLucien**
## Checklist

-   [ ] Tests written for new code (and old code if feasible)
-   [ ] Linter and other CI checks pass
-   [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Type: defect
Notes: duplicates of styles 

I recommend to merge this PR before to merge https://github.com/vector-im/element-web/pull/24877

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by alunturner** [comment]
Hello @JeanPaulLucien , thank you for the contribution.

I see here that you've [linked to another related PR](https://github.com/vector-im/element-web/pull/24877). On the other PR, however, there's [a link to another one of your PRs](https://github.com/matrix-org/matrix-react-sdk/pull/10241) that currently has some discussion in progress about how to proceed.

In order to try and neaten this situation up a bit (and avoid PRs depending on PRs depending on PRs), can I suggest merging the changes here into the PR #24877 as these changes are very much related and I think it makes sense for them to live in one PR. Then we'll just have one PR depending on another, which is a bit easier to track and reduces the chances of us accidentally merging one without the other.

How does that sound to you?

**Comment by JeanPaulLucien** [comment]
@alunturner there is dependency without these changes.

File: res/css/_common.pcss [hunk 1]
```diff
@@ -65,8 +68,7 @@ body {
     font-size: $font-15px;
     background-color: $background;
     color: $primary-content;
-    border: 0px;
```
Comment (alunturner) [inline res/css/_common.pcss:68]: Having looked at the index.html file, I don't see this being declared inline there. I think that we would need to keep this line (unless it's applied somewhere else that I'm missing)

Comment (JeanPaulLucien) [reply res/css/_common.pcss:68]: It's useless string.

**Comment by alunturner** [comment]
Having played about with this a bit it seems that removing the border indeed doesn't make a difference. There are, however, failing snapshots in Percy which suggests to me that perhaps something is having a knock on effect. This PR has been open a little while now, so perhaps `matrix-org:develop` has moved on since you started this work and that's what is causing it.

If you can merge develop into your branch then we can try and get this PR moving again

**Comment by JeanPaulLucien** [comment]
Fill bug report to Percy.

**Comment by JeanPaulLucien** [comment]
I feel strange myself. In the past other Matrix (Vecrtor) participant said to me that CSS analyzer is smart to resolve duplicates, but later I've received a request to change my code, because I have used same tag in two rules.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
@JeanPaulLucien to move your pull request forward:

- Can you please merge all connected pull requests to a single one and add a proper description about what you are going to change and why?
- Also please link dependent pull requests from other repositories (such as Element web).
- Some tests are failing. I would suggest to update your branches (= merge `develop` into them) and then check if the failing tests are legit.

After that please re-request review:

image

**Comment by weeman1337** [comment]
I missed one point. [Please sign-off your changes according to the contribution guideline](https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md#sign-off).

**Comment by andybalaam** [comment]
Currently, it's unclear what the PR is doing, and there is a large list of actions pending on it, so I'm going to close it for now.

**Pull request closed without merging**